deps.edn

```diff
@@ -11,7 +11,7 @@
          com.taoensso/timbre         {:mvn/version "4.10.0"}
          hickory                     {:mvn/version "0.7.1"}
          com.cognitect/transit-cljs  {:mvn/version "0.8.248"}
-         status-im/pluto             {:mvn/version "iteration-4-5"}
+         status-im/pluto             {:mvn/version "iteration-4-6-SNAPSHOT"}
```

Comment:
why SNAPSHOT" ?

Reply:
Yes will change but waiting for some extra fixes. But good point will change in the PR